### DISCUSSION TITLE
Add Bing search engine as option

### DISCUSSION THREAD

**Discussion by 3kbest** [zen-browser/desktop#1106] [Ideas]
It would be nice to have an option to choose Bing as the search engine

![image](https://github.com/user-attachments/assets/205f3d5a-f9ab-47a6-9515-2e5d8895ef36)

**Comment by mr-cheffy** [comment]
Im doing a rewrite of the search engines

**Reply by thibaultmol** [reply]
Is there a place where we can list search engine suggestions right now? (I would like to see kagi.com listed)
